app/views/animals/new.html.erb

```diff
@@ -0,0 +1,22 @@
+
+
+<h1>Add the details of your animal</h1>
+
+<div class="container">
+  <%= simple_form_for(@animal,  html: { class: "form-group w-50 p-5 mx-auto"}) do |f| %>
+    <%= f.input :name %>
+    <%= f.label :species, 'Animal Type' %>
+    <%= f.input :species, as: :string, label: false %>
+    <%= f.input :hourly_rate %>
+    <%= f.input :place_of_origin %>
+    <%= f.input :age %>
+    <%= f.input :category, collection: Animal::CATEGORIES %>
```

Comment:
I think we want to use an f.association here, as I believe that will create a dropdown menu that lets the user pick from one of the predefined animal categories we've created. As a basic input field (I think) it will just be a blank text box they can write their answer into, without knowing that it needs to perfectly match the ones we wrote.